src/server/contest/views.py

```diff
@@ -292,3 +294,15 @@ def submissions_download_response(request, filename, submission_type):
         response['content-Disposition'] = 'attachment;filename='+filename
         return response
 
+
+@login_required
+def problem_file_download(request, problem_id):
+    problem = all_problems.objects.get(problem_id=problem_id)
+    problem_file = problem.problem_file
+
+    with open(str(problem_file), 'rb') as file:
+        response = HttpResponse(file.read())
+        response['content_type'] = 'application/pdf'
+        filename = "Problem_statement_{}.pdf".format(problem_id)
+        response['content-Disposition'] = 'attachment;filename=' + filename
```

Comment:
Use inline instead of attachment to open document in the browser. 
Make it open in a new tab.